output/eu_crude/analysis.py

```diff
@@ -0,0 +1,153 @@
+import sqlite3 as lite
+import sys
+import matplotlib as plt
+from pyne import nucname
+
+# tells command format if input is invalid
+if len(sys.argv) < 2:
+    print('Usage: python analysis.py [cylus_output_file]')
+
+
+def analysis(filename):
+    """ does simple analysis of cyclus input file.
+
+    Parameters
+    ----------
+    filename: int
+        cyclus output file to be analysed.
+
+    Returns
+    -------
+    prints on terminal total snf and isotope mass.
+
+    """
+
+    sink_id = get_sink_agentIds()
+    resources = cur.execute(exec_string(sink_id, 'transactions.receiverId', '*')).fetchall()
+    snf_inventory = cur.execute(exec_string(sink_id, 'transactions.receiverId', 'sum(quantity)')).fetchall()[0][0]
+    waste_id = get_waste_id(resources)
+    inven = isotope_calc(waste_id, snf_inventory)
+    print('Total snf mass is = ' + str(snf_inventory) + ' kg')
+    print(inven)
+
+
+def get_sink_agentIds():
+    """ Gets all sink agentIds from Agententry table.
+
+        Agententry table has the following format:
+            SimId / AgentId / Kind / Spec / Prototype / ParentID / Lifetime / EnterTime
+
+    Paramters
+    ---------
+    none
+
+    Returns
+    -------
+    sink_id: array
+        array of all the sink agentId values.
+
+    """
+
+    sink_id = []
+    agent = cur.execute("select * from agententry where spec like '%sink%'")
+
+    for ag in agent:
+        sink_id.append(ag[1])
+    return sink_id
+
+
+def get_waste_id(resource_array):
+    """ Gets waste id from a resource array
+
+    Paramters
+    ---------
+    resrouce_array: array
+        array fetched from the resource table.
+
+    Returns
+    -------
+    waste_id: array
+        array of qualId for waste
+
+    """
+
+    wasteid = []
+
+    # get all the wasteid
+    for res in resource_array:
+        wasteid.append(res[7])
+
+    # make it a set
+    return set(wasteid)
+
+
+def exec_string(array, search, whatwant):
+    """ Generates sqlite query command for various purposes.
```

Comment:
The command has only one purpose, actually. This sql command can only select things an inner join between resources and transactions on the resource id. Be specific.